modules/logging/filters.go

```diff
@@ -185,15 +187,113 @@ func (m IPMaskFilter) Filter(in zapcore.Field) zapcore.Field {
 	return in
 }
 
+type ActionType string
+
+const (
+	ReplaceType ActionType = "replace"
+	DeleteType  ActionType = "delete"
+)
+
+type cookieFilterAction struct {
+	Type  ActionType `json:"type"`
+	Name  string     `json:"name"`
+	Value string     `json:"value,omitempty"`
```

Comment:
Could use godoc comments on these fields as well (they get pulled into the JSON docs)